multi-checker/script/utils.py

```diff
@@ -82,3 +82,31 @@ def isDebug():
 def debug(msg):
     if _DEBUG:
         print(f'[DEBUG] {msg}')
+
+def findDirMatchingAux(root: str, stopCondition: Callable) -> Optional[str]:
+    if isFile(root):
+        return None
+    if stopCondition(root):
+        return root
+    for x in ls(root):
+        result = findDirMatchingAux(x, stopCondition)
```

Comment:
only descend recursively if root is a wrapper dir. For example, you do not want to descend in the following case:

root/A
root/B

Reply:
This should be fixed with 8140d9dcbca827ba3c592f320c5ead875b561e00.